types/dailymotion.d.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseReactPlayer, BaseReactPlayerProps } from './_utils'
+
+export interface DailyMotionConfig {
+  params?: Record<string, any>
+}
+
+export interface DailyMotionPlayerProps extends BaseReactPlayerProps {
```

Comment:
To the best of my understanding, you don't have to extend `DailyMotionPlayerProps` from `BaseReactPlayerProps` because in the implementation of BaseReactPlayer, every given Props are being extended automatically.
```js
export class BaseReactPlayer<T extends BaseReactPlayerProps> extends Component<T, any> {
}
```
any interface provided to BaseReactPlayer will extends BaseReactPlayerProps automatically, you don't have to extend them manually.

Reply:
That's intentional. If someone wants the props of the (for example) `DailyMotionPlayer`, they will do `import DailyMotionPlayer, { DailyMotionPlayerProps } from 'react-player/dailymotion'`

Reply:
that's a fair point :100: